code-challenges.test.js

```diff
@@ -14,33 +14,215 @@
 
+
+
+// const taker=(array)=>{
+//   return array.map((value)=>{"${Object.value[0]} is a ${Object.value[1]}"})
+// }
+// - Expected  - 3
+// + Received  + 3
+
+// Array [
+// -   "Ford Prefect is a hitchhiker.",
+// -   "Zaphod Beeblebrox is president of the galaxy.",
+// -   "Arthur Dent is a radio employee.",
+// +   undefined,
+// +   undefined,
+// +   undefined,
+// ]
+
+// const taker=(array)=>{
+//   return array.map((value)=>{"${Object.values(value)}"})
+// }
+// - Expected  - 3
+// + Received  + 3
+
+// const taker=(array)=>{
+//   return array.map((value)=>{Object.values(value)})
+// }
+// - Expected  - 3
+// + Received  + 3
+
+// const taker=(array)=>{
+//  return array.map(value)=>{(value)})}
+//  - Expected  - 3
+//  + Received  + 3
+
+// const taker=(array)=>{
+//   return `${array[0].name} is a ${array[0].occupation}`}
+//   - Expected  - 3
+//   + Received  + 3
+
+// const taker=(array)=>{
+// return array.map((value)=>{`${Object.values[value][0]} is a ${Object.values[value][1]}`})}
+// - Expected  - 3
+// + Received  + 3
+
+// const taker=(array)=>{
+//   return (`${array[0].name} is ${array[0].occupation}`)}
+// Test Suites: 1 failed, 1 total
+// Tests:       1 failed, 1 total
+
+// const taker=(array)=>{
+// let obj = Object.values(array)
+// let newObj =[]
+// for(let i=0;i<obj.length;i++){
+// newObj.push(`${obj[i].name.slice(0,1).toUpperCase()+obj[i].name.slice(1,obj[i].name.length)
+// } is ${obj[i].occupation}`)}
+// return newObj
+// }
+
+//create a function called taker
+  //function will take in the parameter of an array
+    //The function will access the value of each key value pair within the object ny the variable obj.
+    //we will be able to access the values of obj using dot notation.
+    //create a new variable of newObj with an empty array to push strings into
+      //create a for loop to run through each index of the obj variable
+      //in the for loop create a new variable called newStr
+        //newStr will take in the obj at an index and the name using dot notation
+          //newStr will make the name into an array and capitalize the starting letter of the string using built methods map, to uppercase, split, substring, lowercase and join
+    //the function will string interpolate the newStr variable and obj at an index and dot notation to create a string
+    //moving the string interpolation into the newObj using built in method push
+  //The function will return newObj with all the strings in an array
+//verify with yarn jest
+
+// const taker=(array)=>{
+//   let obj = Object.values(array)
+//   let newObj =[]
+//   for(let i=0;i<obj.length;i++){
+//     const newStr = obj[i].name.split(' ').map(value=>value[0].toUpperCase()+value.substring(1).toLowerCase()).join(' ')
+//   newObj.push(`${newStr} is ${obj[i].occupation}.`)}
+//   return newObj
+//   }
+
+//   Test Suites: 1 passed, 1 total
+// Tests:       3 passed, 3 total
+
+const taker=(array)=>{
```

Comment:
Nice work!  I appreciate you leaving your previous attempts and updated refactor code in here.  I saw a few attempts previously using higher order functions, keep working on utilizing them instead of for loops whenever possible.